### PR TITLE
963 remove items from unfpa dashboard

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
[963 UNFPA Dashboard Change: Please remove SAYANA Press from all visuals](https://github.com/beyondessential/tupaia-backlog/issues/963)

### Changes:
Add migration to remove these items from all visuals

### CODE REVIEW

File: packages/database/src/migrations/20201111013312-RemoveItemsFromLineChartsInUnfpaDashboard-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    },
+  },
+};
+
+// Table name = 'Reproductive Health Stock Status: Months Of Stock (mSupply)'
+const UNFPARhProductsMosDashboardReport = {
+  id: 'UNFPA_RH_Products_MOS',
+  targetItems: {
+    UNFPA_RH_SAYANA_Press_MOS: {
+      codes: ['MOS_4752843e'],
```
Comment (IgorNadj) [inline packages/database/src/migrations/20201111013312-RemoveItemsFromLineChartsInUnfpaDashboard-modifies-data.js:100]: I suggest breaking these up into dataBuilderConfig and viewJson for clarity. I figured it out when I scrolled down to the implementation but it took me a minute, at first I thought one of us might have had old data in their database and I went to check.

**Comment by IgorNadj** [comment]
Just the one comment, apart from that looks good!

File: packages/database/src/migrations/20201111013312-RemoveItemsFromLineChartsInUnfpaDashboard-modifies-data.js (hunk 1 above)
Comment (billli0) [reply packages/database/src/migrations/20201111013312-RemoveItemsFromLineChartsInUnfpaDashboard-modifies-data.js:100]: Hi Igor, thanks for reviewing this. Sorry about the one minute headache💆🏻‍♂️. I will break it up now.

**Review by IgorNadj — APPROVED** [review]
LGTM 🏌️‍♀️

**Pull request merged**